tgui/packages/tgui/interfaces/RCD.js

```diff
@@ -95,6 +95,11 @@ const AirlockSettings = (props, context) => {
             icon="pen-alt"
             content="Edit"
             onClick={() => modalOpen(context, 'renameAirlock')}
+          /><Button
```

Comment:
```suggestion
          />
          <Button
```
Formatting thing. Also stylistically with the UI here, you really shouldn't be putting these buttons on the same LabeledListItem b/c the edit button above will move around when you're editing the name of the airlock. Please just put it on it's own line with another `labeledlist.item` block.